src/lib/comms/include/sol-bluetooth.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2016 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <sol-network.h>
+
+enum sol_bt_uuid_type {
+	SOL_BT_UUID_TYPE_16 = 2,
+	SOL_BT_UUID_TYPE_32 = 4,
+	SOL_BT_UUID_TYPE_128 = 16,
+};
+
+struct sol_bt_uuid {
+	enum sol_bt_uuid_type type;
+	union {
+		uint16_t val16;
+		uint32_t val32;
+		uint8_t val128[16];
+		uint8_t val[0];
+	};
+};
+
+int sol_bt_uuid_from_str(struct sol_bt_uuid *uuid, const char *str);
```

Comment:
are these always null-terminated? if not then we should use a str_slice, that have an easy conversion from char* using `sol_str_slice_from_str()`